platforms-serverless/vercel-cli/test.sh

```diff
@@ -2,4 +2,11 @@
 
 set -eu
 
-npx ts-node ../../utils/fetch-retry-and-confirm-version.ts --url https://e2e-vercel-api.vercel.app/api --prisma-version $(sh ../../utils/prisma_version.sh) --binary-string ',"files":["index.js","package.json","query-engine-rhel-openssl-1.0.x","schema.prisma"]'
+# checks whether PRISMA_FORCE_NAPI has length equal to zero
+if [[ -z "${PRISMA_FORCE_NAPI+x}" ]]; then
+  files=',"files":["index.js","package.json","query-engine-rhel-openssl-1.0.x","schema.prisma"]'
+else
+  files=',"files":["index.js","libquery_engine_napi-rhel-openssl-1.0.x.so.node","package.json","schema.prisma"]'
+fi
+
+npx ts-node ../../utils/fetch-retry-and-confirm-version.ts --url https://e2e-vercel-api.vercel.app/api --prisma-version $(sh ../../utils/prisma_version.sh) --binary-string $files
```

Comment:
Can we also get the deployment URL to use here?

Reply:
I tried using it but for some reason it did not work